extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/config/VaultSecretConfigKvPrefixPathsConfig.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.vault.runtime.config;
+
+import java.util.List;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class VaultSecretConfigKvPrefixPathsConfig {
```

Comment:
This class is no longer used, right? We can remove this or and keep https://github.com/quarkusio/quarkus/pull/13300/files#diff-ccdd86f7203916d7b0e179ba75491f1b532fe12f8c747f676b28ef0a05950420R263 with proper renaming or vice versa.

Reply:
you are correct. I have cleaned up the obsolete code. thanks